src/Modal.js

```diff
@@ -267,6 +268,15 @@ const Modal = React.createClass({
       );
     }
 
+    const keys = Object.keys(_props);
+    const propTypes = Object.keys(Modal.propTypes);
+    const newProps = {};
+    keys.map(function (prop) {
+      if (!propTypes.includes(prop)) {
```

Comment:
Hey @taion, great catch. I didn't use `_.omit` because lodash is just a dev dependency in this project, and I didn't want to add it just for this. If ya'll are ok with adding it, i'll happily change the code.